client/ci-build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
if you use `#!/usr/bin/env bash` like the other script you can use the keyword `function`. I guess we should standardize to use bash, right?

Reply:
Bash was/is not available in some of the prod containers

Reply:
Ignore the previous comment